services/media/service/src/common/utils/token-utils.ts

```diff
@@ -1,23 +1,33 @@
 import {
   generateLongLivedToken,
   getServiceAccountToken,
-  TokenResult,
 } from '@axinom/mosaic-id-link-be';
+import NodeCache from 'node-cache';
```

Comment:
Is there/can we have a check if we get an authorization error that we try (once) to get a new token? Like if the secret key was regenerated that we do not have to wait for the cache to expire.

Reply:
There is no good way for now, but we already discussed this with Ruwan and came up with an appropriate solution which requires extra changed in ID libs. Will update this comment with the task number when I get it.